INSTALL.md

```diff
@@ -16,9 +16,9 @@ should grab it automatically into the local `src/` folder.
 
 ## Installing
 
-You need an SD card, format it msdos, then partition it:
+You need an SD card, create a empty msdos table, then partition it:
 
-* `/boot`, fat32, 500ish MB
+* `/boot`, fat32 (type b), 500ish MB
```

Comment:
What does this mean?

In theory this partion is whatever uboot can read, if we wanted to I could enable the config for ext2 as well if wanted.

I know mister uses the main exfat partition for this and uboot is fine with it.

Reply:
You wrote fat32, so it is type b in fdisk. uboot supports ext2 and, probably exfat.

Reply:
Gotcha! I haven't been using fdisk for that part so I missed the significance of it.